cmd/export/cluster.go

```diff
@@ -0,0 +1,221 @@
+package export
+
+import (
+	"fmt"
+	"strings"
+
+	authv1 "github.com/openshift/api/authorization/v1"
+	securityv1 "github.com/openshift/api/security/v1"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/exp/slices"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type ClusterScopeHandler struct {
+}
+
+var admittedClusterScopeKinds = []string{"ClusterRole", "ClusterRoleBinding", "SecurityContextConstraints"}
+
+func NewClusterScopeHandler() *ClusterScopeHandler {
+	clusterScopeHandler := &ClusterScopeHandler{}
+
+	return clusterScopeHandler
+}
+
+func (c *ClusterScopeHandler) filterRbacResources(resources []*groupResource, log logrus.FieldLogger) []*groupResource {
+	log.Debug("Looking for ServiceAccount resources")
+
+	handler := NewClusterScopedRbacHandler(log)
+	for i := len(resources) - 1; i >= 0; i-- {
```

Comment:
The reason for using reverse loop is that we remove the resources of the admitted kinds (L41) which alters the content of the initial array: the reverse navigation avoids to skip some of the elements because the elements with index from 0-i always remain unchanged.
to use range I think we need to create a temporary slice and populate it by copying the elements. if you agree, I can do that way (should be a copy of references,  so not a big issue in terms of memory)

Reply:
Resolved in latest commit